client/src/ui/LoadStatus.res

```diff
@@ -0,0 +1,21 @@
+// LoadStatus is a value to represent that there is or was a loading action.
```

Comment:
```suggestion
// LoadStatus is a value to represent that something is being or has been loaded.
```
I found the wording slightly awkward. I'm not sure if my suggestion is an improvement.